android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -937,6 +979,19 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
         }
     }
 
+    fun requestScreenShare(data: Intent?) {
+        hmssdk.startScreenshare(object : HMSActionResultListener {
```

Comment:
should return `result.success(nil)` if successful or exception hashmap on error
use the existing result action listener